scootapi/setup/api.go

```diff
@@ -26,17 +26,19 @@ type ApiConfig struct {
 
 // LocalApiStrategy starts up a local apiserver
 type LocalApiStrategy struct {
-	apiCfg  *ApiConfig
-	builder Builder
-	cmds    *Cmds
+	apiCfg   *ApiConfig
+	builder  Builder
+	cmds     *Cmds
+	logLevel log.Level
 }
 
 // Create a new Local ApiServer that will serve the bundlestore api, using builder and cmds to start
-func NewLocalApiStrategy(apiCfg *ApiConfig, builder Builder, cmds *Cmds) *LocalApiStrategy {
+func NewLocalApiStrategy(apiCfg *ApiConfig, builder Builder, cmds *Cmds, level log.Level) *LocalApiStrategy {
 	return &LocalApiStrategy{
-		apiCfg:  apiCfg,
-		builder: builder,
-		cmds:    cmds,
+		apiCfg:   apiCfg,
+		builder:  builder,
+		cmds:     cmds,
+		logLevel: level,
```

Comment:
For all of these strategies, can you move "level" into the higher level *Config struct and then access it as localApiStrategy.apiCfg.level?